src/nvim/eval/funcs.c

```diff
@@ -3507,6 +3507,29 @@ static void f_getloclist(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   get_qf_loc_list(false, wp, &argvars[1], rettv);
 }
 
+
+/// "getmarklist()" function
+static void f_getmarklist(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+{
+  buf_T *buf = NULL;
+
+  if (tv_list_alloc_ret(rettv, kListLenMayKnow) == NULL) {
```

Comment:
This is never NULL.

Reply:
should be fixed in 67640865f93e38be68fd69c2c7b04970e7da14b6